library/src/main/java/com/datatorrent/lib/io/jms/JMSTransactionableStore.java

```diff
@@ -175,7 +178,7 @@ public void connect() throws IOException
       }
 
       Queue queue = getBase().getSession().createQueue(queueName);
-      QueueBrowser browser = getBase().getSession().createBrowser(queue);
+      QueueBrowser browser = getBase().getSession().createBrowser(queue,"operatorId=" + String.valueOf(getOperatorId()));
```

Comment:
String.valueOf is unnecessary. So the following should suffice:

"operatorId=" + getOperatorId()